relational-operators: Recommend to use consistent relational operators (#34)
leapfrogtechnology/coding-guidelines: docs/introduction.md

* Follow Gitflow.

* Sanity checklist document should be present.

* Consistent Relational Operators:

>   * Recommended to use either `>=` / `<=` or `>` / `<` relationals operators consistently in entire codebase.

Reviewer: ```suggestion
  * Use relational operators either `>=` / `<=` or `>` / `<`  consistently in entire codebase.
```

>   * Scenario: In front end, there are logics like `>=256 && <=512` and in other components/files or in backend  there is `>255 && <511`.

Reviewer: ```suggestion
  * Example: In some places, there are `>=256 && <=512` and in other place there is `>255 && <511`.
```

>   * Benefits: No high stress to human mind. Searching for those numbers in entire codebase will result accurate.

Reviewer: ```suggestion
  * Why? Lesser cognitive burden, enforce consistency and avoid confusion. 
```
